integrationservertest/logs_filters.go

```diff
@@ -102,12 +102,18 @@ var (
 		},
 	})
 
-	syncSourcemapContextCanceled = apmErrorLog(types.Query{
+	initialSearchQueryContextCanceled = apmErrorLog(types.Query{
 		MatchPhrase: map[string]types.MatchPhraseQuery{
 			"message": {Query: "failed to sync sourcemaps metadata: failed to run initial search query: fetcher unavailable: context canceled"},
 		},
 	})
 
+	scrollSearchQueryContextCanceled = apmErrorLog(types.Query{
+		MatchPhrase: map[string]types.MatchPhraseQuery{
```

Comment:
Should we think of excluding all "context canceled" logs that we observe? I expect there are multiple occasions where this happens on shutdown and filtering them one by one manually seems to require a lot of manual intervention. This is for a follow-up, worth a separate issue.